File: Chapter 2/AverageAndGrade.java
import java.util.Scanner;

public class AverageGrade {

public static void main (String[]args) {

Scanner input = new Scanner(System.in);

System.out.print("Enter first score :");
double firstScore = input.nextDouble();

System.out.print("Enter second score :");
double secondScore = input.nextDouble();

System.out.print("Enter third score :");
double thirdScore = input.nextDouble();

double average = (firstScore + secondScore + thirdScore) / 3;

System.out.println("The average score is " + average);

if (average >= 90 && average <= 100) {
System.out.println("A");
} else if (average >= 80 && average < 90) {
System.out.println("B");
} else if (average >= 70 && average < 80) {
System.out.println("C");
} else if (average >= 60 && average < 70) {
System.out.println("D");
} else if (average >= 0 && average < 60) {
System.out.println("F");
} 

}

}

File: Chapter 2/average_and_grade.py
first_score = float(input("Enter first score: "))
second_score = float(input("Enter second score: "))
third_score = float(input("Enter third score: "))


average = (first_score + second_score + third_score) / 3

print("Average Score:", average)

if average >= 90 and average <= 100:
	print('A')
elif average >= 80 and average < 90:
	print('B')
elif average >= 70 and average < 80:
	print('C')
elif average >= 60 and average < 70:
	print('D')
elif average >= 0 and average < 60:
	print('F')





